components/input/Input .tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import styles from "./Input.module.css";
+
+const [name, setName] = useState("");
```

Comment:
Not sure, but I think this should be inside the Input() function?

Reply:
good hint, thanks :-)